public/index.js

```diff
@@ -0,0 +1,20 @@
+const items = querySelector(.'todos')
+
+cells.forEach((cell) => {
+
+})
+
+})
```

Comment:
I recommend commenting out unfinished code to let your reviewer know that its not expected to be functional yet. Maybe even leave a comment above it describing what it will do when finished above.